### PR TITLE
Updated and Added More Issue Templates

### LINKED ISSUE #75: Update and Add More Issue Templates
Should I work on this?

### PR DESCRIPTION
**Pull request by MrKrishnaAgarwal**
## Describe your changes

I have Updated and Added More Issue Templates, and converted .md to .yml with better choices

## Screenshots - If Any (Optional)

image
image
image

## Issue ticket number and link - If Any

Closes #75 

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] Followed the repository's [Contributing Guidelines](/CONTRIBUTING.md).
- [x] I ran the app and tested it locally to verify that it works as expected.

## Additional Information (Optional)

cc @EmmaDawsonDev 

<a href="https://gitpod.io/#https://github.com/AccessibleForAll/AccessibleWebDev/pull/76"></a>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: "Description"
+      description: Please enter an explicit description of your issue
+      placeholder: Short and explicit description of your incident...
+    validations:
+      required: true
+  - type: input
```
Comment (EmmaDawsonDev) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:16]: This one can be removed.

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    id: reprod-url
+    attributes:
+      label: "Reproduction URL"
+      description: Please enter your GitHub URL to provide a reproduction of the issue
+      placeholder: ex. https://github.com/USERNAME/REPO-NAME
+    validations:
+      required: true
```
Comment (EmmaDawsonDev) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:23]: This whole section can be removed, it's always going to affect the same URL

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      label: "Reproduction steps"
+      description: Please enter an explicit description of your issue
+      value: |
+        1. Go to '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. See error
+      render: bash
+    validations:
+      required: true
```
Comment (EmmaDawsonDev) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:36]: Remove the required here, it's not always possible to provide reproduction steps

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+labels: [
+  "bug"
+]
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: "Description"
+      description: Please enter an explicit description of your issue
+      placeholder: Short and explicit description of your incident...
```
Comment (EmmaDawsonDev) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:13]:
```suggestion
      placeholder: Detailed description of your issue...
```

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+title: "🐛 [BUG] - <title>"
+labels: [
+  "bug"
+]
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: "Description"
+      description: Please enter an explicit description of your issue
```
Comment (EmmaDawsonDev) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:12]:
```suggestion
      description: Please enter a detailed description of your issue
```

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml (hunk 2 above)
Comment (EmmaDawsonDev) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:23]:
```suggestion
  
```

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+      label: "Reproduction URL"
+      description: Please enter your GitHub URL to provide a reproduction of the issue
+      placeholder: ex. https://github.com/USERNAME/REPO-NAME
+    validations:
+      required: true
+  - type: textarea
+    id: reprod
+    attributes:
+      label: "Reproduction steps"
+      description: Please enter an explicit description of your issue
```
Comment (EmmaDawsonDev) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:28]:
```suggestion
      description: Please enter the steps you took when the issue happened
```

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+  - type: dropdown
+    id: os
+    attributes:
+      label: "OS"
+      description: What is the impacted environment ?
+      multiple: true
+      options:
+        - Windows
+        - Linux
+        - Mac
```
Comment (EmmaDawsonDev) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:78]: Can we also add iOS, android and other to this section?

File: .github/ISSUE_TEMPLATE/FEATURE-REQUEST.yml [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+    validations:
+      required: false
+  - type: textarea
+    id: reference_issues
+    attributes:
+      label: "Reference Issues"
+      description: Common issues
+      placeholder: "#Issues IDs"
+    validations:
+      required: false
```
Comment (EmmaDawsonDev) [inline .github/ISSUE_TEMPLATE/FEATURE-REQUEST.yml:31]: Remove these three inputs.

File: .github/ISSUE_TEMPLATE/FEATURE-REQUEST.yml [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+    validations:
+      required: true
+  - type: textarea
+    id: basic_example
+    attributes:
+      label: "Basic Example"
+      description: Indicate here some basic examples of your feature.
+      placeholder: A few specific words about your feature request.
+    validations:
+      required: true
```
Comment (EmmaDawsonDev) [inline .github/ISSUE_TEMPLATE/FEATURE-REQUEST.yml:47]:
```suggestion
      required: false
```

File: .github/ISSUE_TEMPLATE/FEATURE-REQUEST.yml [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+  - type: textarea
+    id: unresolved_question
+    attributes:
+      label: "Unresolved questions"
+      description: What questions still remain unresolved ?
+      placeholder: Identify any unresolved issues.
+    validations:
+      required: false
```
Comment (EmmaDawsonDev) [inline .github/ISSUE_TEMPLATE/FEATURE-REQUEST.yml:63]: Remove these ones

File: .github/ISSUE_TEMPLATE/OTHER.yml [hunk 11]
```diff
@@ -0,0 +1,22 @@
+name: Other
+description: Describe anything here.
+title: Other <subject>
```
Comment (EmmaDawsonDev) [inline .github/ISSUE_TEMPLATE/OTHER.yml:3]:
```suggestion
title: [OTHER] <subject>
```

Let's use the same format for all templates

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 12]
```diff
@@ -0,0 +1,80 @@
...
+        - Safari
+        - Microsoft Edge
+        - Opera
+    validations:
+      required: false
+  - type: dropdown
+    id: os
+    attributes:
+      label: "OS"
+      description: What is the impacted environment ?
```
Comment (EmmaDawsonDev) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:73]:
```suggestion
      description: What is the impacted operating system?
```

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
There's just a few updates to make these easier to use for people. Mostly I don't think we need so many questions and most of them don't need to be required. Let me know if you have any questions.

**Comment by MrKrishnaAgarwal** [comment]
@EmmaDawsonDev I have made all the Required Changes

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 13]
```diff
@@ -0,0 +1,80 @@
+name: "🐛 Bug Report"
+description: Create a new ticket for a bug.
+title: "🐛 [BUG] - <title>"
+labels: [
+  "bug"
+]
+body:
+  - type: textarea
```
Comment (EmmaDawsonDev) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml:8]: I think there's something slightly wrong with the formatting. I'm seeing the following error when I try to view the file:
image

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/common-validation-errors-when-creating-issue-forms

File: .github/ISSUE_TEMPLATE/FEATURE-REQUEST.yml (hunk 10 above)
Comment (EmmaDawsonDev) [reply .github/ISSUE_TEMPLATE/FEATURE-REQUEST.yml:39]: Remove Drawbacks and unresolved questions too please

**Review by EmmaDawsonDev — COMMENTED** [review]
There seems to be a couple of issues with the syntax now, probably just some wrong indentation somewhere.

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
Changes requested in previous comment (forgot to check correct option)

**Pull request closed without merging**